pkg/gitreceive/build.go

```diff
@@ -256,8 +263,9 @@ func build(conf *Config, s3Client *s3.S3, builderKey, gitSha string) error {
 	if !usingDockerfile {
 		buildHook.Dockerfile = ""
 	} else {
+		json.Unmarshal([]byte("{}"), &procType)
```

Comment:
`procType = pkg.ProcessType{}` seems semantically equivalent here, and will be more performant